gst/nnstreamer/tensor_source/tensor_src_iio.c

```diff
@@ -681,13 +681,15 @@ gst_tensor_src_iio_get_float_from_file (const gchar * dirname,
 {
   gchar *filename, *filepath, *file_contents = NULL;
 
+  errno = 0;
   filename = g_strdup_printf ("%s%s", name, suffix);
   filepath = g_build_filename (dirname, filename, NULL);
 
   if (!g_file_get_contents (filepath, &file_contents, NULL, NULL)) {
     GST_INFO ("Unable to retrieve data from file %s.", filename);
   } else {
-    if (!sscanf (file_contents, "%f", value)) {
+    *value = (gfloat) g_ascii_strtod (file_contents, NULL);
```

Comment:
@kparichay  I have a trivial request. Could you archive the report result of SVACE in the commit body?

Reply:
updated.